File: material-stepper/src/test/java/com/stepstone/stepper/test/TestApplication.java
package com.stepstone.stepper.test;

import android.app.Application;

import org.mockito.MockitoAnnotations;
import org.robolectric.TestLifecycleApplication;

import java.lang.reflect.Method;

/**
 * @author Piotr Zawadzki
 */
public class TestApplication extends Application implements TestLifecycleApplication {

        @Override
        public void beforeTest(Method method) {
        }

        @Override
        public void prepareTest(Object test) {
            MockitoAnnotations.initMocks(test);//initialize all the mocks that were specified using annotations
        }

        @Override
        public void afterTest(Method method) {
        }

}

File: material-stepper/src/test/java/com/stepstone/stepper/test/TestApplication.kt
package com.stepstone.stepper.test

import android.app.Application

import org.mockito.MockitoAnnotations
import org.robolectric.TestLifecycleApplication

import java.lang.reflect.Method

/**
 * @author Piotr Zawadzki
 */
class TestApplication : Application(), TestLifecycleApplication {

    override fun beforeTest(method: Method) {}

    override fun prepareTest(test: Any) {
        MockitoAnnotations.initMocks(test)//initialize all the mocks that were specified using annotations
    }

    override fun afterTest(method: Method) {}

}